src/docs/release/breaking-changes/scaffold-messenger.md

```diff
@@ -95,6 +96,22 @@ Typically, the ScaffoldMessenger widget is introduced by the MaterialApp
 at the top of your application widget tree.
 ```
 
+The `ScaffoldMessenger` will also assert if attempting to show `SnackBar`s to nested `Scaffold`s. If
+a nested set of `Scaffold`s were to share the same `ScaffoldMessenger`, then duplicate `SnackBar`s
```

Comment:
=> If a nested set of Scaffolds share the same ScaffoldMessage, then duplicate SnackBars might appear in your UI and you would see the following assertion error:

...  (I'd love to update that error message, too.  :) )

To avoid this situation, place a ScaffoldMessenger between the SnackBar and the Scaffold.

Reply:
Happy to update the error message, it has not landed yet (https://github.com/flutter/flutter/pull/70227). What should it say?